docs/source/models/cause.rst

```diff
@@ -275,14 +281,125 @@ Birth Prevalence
 Cause Model Transitions
 +++++++++++++++++++++++
 
-Incidence
-^^^^^^^^^
+.. todo::
+
+	#. Add progression transitions, deterministic transitions, and severity splits to summary table above? (should this be the case? or should these only be discusse din the transition section? I am thinking the latter)
+	#. Enhance blurb to beginning of cause model transition section about how we use probabilies to inform cause model transitions (to come in next commits)
+	#. Detail incidence, remisison, and duration-based transition sections (to come in next commits)
+	#. Detail progression transitions, deterministic transitions, and severity splits (to come in future PRs)
+
+Vivarium uses probabilities to make decisions about how and when simulants 
+move between cause model states. 
+
+Incidence Rates
+^^^^^^^^^^^^^^^
+
+Generally, incidence is a measure of new cases of a given condition that occur 
+in a given timeframe and population. The count value of new cases of the 
+condition of interest will always be the numerator of incidence measures. The 
+denominator of incidence measures is somewhat more complex and is critical to 
+ensuring an accurate data source to inform cause model transition rates. 
+
+**Cumulative incidence** is a specific type of incidence measure that can be 
```

Comment:
Aha! I figured out the relationship between the incidence rate and the hazard rate function, using the observation that the person-time of the at-risk population is the initial population times the area under the survival function curve, and the fact that the hazard function h(t) = -S'(t) / S(t).

Namely, I think the standard definition of incidence rate is (incident cases) / (person-time of at-risk population), and the following are equivalent characterizations:

incidence rate over an interval [t_1, t_2]
= (incident cases in [t_1 ,t_2]) / (person-time of at-risk population during [t_1, t_2])
= (decrease in S(t) from t_1 to t_2) / (area under S(t) from t_1 to t_2)
= - (S(t_2) - S(t_1)) / ((average fraction of population at risk during [t_1, t_2]) * (t_2 - t_1))
= (average rate of decrease of at-risk population) / (average at-risk population during [t_1, t_2])
= (integral of h(t)*S(t) from t_1 to t_2) / (integral of S(t) from t_1 to t_2)
= population-weighted average of h(t) on the interval [t_1, t_2]

In particular, the incidence rate is **not** equal to what I called lambda_t above (the unweighted average of h(t) on [t_1, t_2]), unless the hazard rate is constant.

I think these two different averages should have the following interpretations (but I haven't thought about it carefully):

- The unweighted average hazard rate should be the constant hazard rate that results in the same decrease in S(t) over the interval [t_1, t_2] if you started with equal at-risk populations.

- The population-weighted hazard rate (i.e. the incidence rate) should be the constant hazard rate that results in the same decrease in S(t) over the interval [t_1, t_2] if the two at-risk populations had the same person-time during the interval.

Anyway, not all of this should go on the page about incidence rates, but some of it should go on the page about survival analysis.

Reply:
In the most recent commits, I changed "cumulative incidence" to what we discussed as "total population incidence rate" and "susceptible population incidence rate". So far, I have focused on defining what these are and how they may be used in our model.

I am planning to get into how the susceptible population incidence rate and hazard rate relate in the limitation section (currently just a to-do still on this page) and in the linked survival analysis page.

Let me know if you think more of the information from these comments belongs in the existing text of this PR instead of the to-do limitation and survival analysis sections and I can work on adding it in. Otherwise, this will be helpful moving forward onto those sections, so thank you!